app/models/realm.rb

```diff
@@ -34,6 +34,11 @@ class Realm < ApplicationRecord
     end
   }
 
+  apipie :class, desc: "A class representing #{model_name.human} object" do
+    sections only: %w[all additional]
+    prop_group :basic_model_props, ApplicationRecord, meta: { example: 'EXAMPLE.COM' }
+    property :realm_type, String, desc: 'Realm type, e.g. FreeIPA or Active Directory'
```

Comment:
what about id and name?

Reply:
@shiramax , they come from https://github.com/theforeman/foreman/blob/2cd4d88297a36d9a43a34a64badb3b7aa18e7d31/app/models/application_record.rb#L6-L11

If just like `param_group` for APIs.